test/omitIndexes.js

```diff
@@ -0,0 +1,22 @@
+import RA from '../src/index';
+import eq from './shared/eq';
+
+
+describe('omitIndexes', function() {
+  it('tests currying', function () {
+    const indexes = [1, 3];
+    const arr = ['a', 'b', 'c', 'd'];
+    const expected = ['a', 'c'];
+
+    eq(RA.omitIndexes(indexes, arr), expected);
+    eq(RA.omitIndexes(indexes)(arr), expected);
+  });
+
+  it('tests non existing indexes are omitted', function() {
+    const indexes = [-1, 1, 3];
+    const arr = ['a', 'b', 'c', 'd'];
+    const expected = ['a', 'c'];
+
+    eq(RA.omitIndexes(indexes, arr), expected);
+  });
```

Comment:
Pls add tests for 3 more usecases:
 - empty indexes (will return unmodified list)
 - empty list (will return empty list)
 - combination of empty list and empty indexes

Just to be sure we have corner cases covered. I will adjust `pickIndexes` in https://github.com/char0n/ramda-adjunct/issues/181 to be consistent with your `omitIndexes`.

Then we can merge.